tools/quantwiz_iq/quantwiz_iq.xml

```diff
@@ -0,0 +1,209 @@
+<tool id="quantwiz_iq" name="QuantWiz-IQ" version="2.0.1">
+    <description>Isobaric Quantitation using QuantWiz-IQ</description>
+    <requirements>
+        <requirement type="package" version="2.0">quantwiz-iq</requirement>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+cp '$input_file' '$input_file.name'
+&& cat '$new_param_file' > '$iq_params'
+#if $input_purity_file.input_purity_file_option == "history":
+&& cp '$input_purity_file.purity_type' '$input_purity_file.purity_type.name'
+&& echo "Purity_corrections_values=$input_purity_file.purity_type.name" >> '$iq_params'
+#else:
+&& PROG_PATH=`which QuantWiz_IQ`
+&& cp \${PROG_PATH%/*}/Purity_correction/$input_purity_file.purity_type .
```

Comment:
@pravs3683 can you explain this here? What do you wanna do?

Reply:
I am copying a "purity correction" file to the working directory. These are some default files provided in the QuantWiz-IQ tool.
This file is shared in the "$PREFIX/bin/Purity_correction" folder of the QuantWiz-IQ's Bioconda recipe. This piece was suggested by @jj-umn and it works perfectly.